docs/_data/components/schemas/forecast-limit-period.yaml

```diff
@@ -21,21 +21,38 @@ forecast-period-proposal:
 
   allOf:
     - $ref: '#/period'
-    - $ref: ./limit-data.yaml
+    - $ref: ./limit-proposal.yaml
 
-forecast-period-snapshot-slim:
+proposal-considered:
+  type: object
   allOf:
-    - $ref: '#/forecast-period-proposal'
+    - $ref: '#/proposal'
+    - type: object
+      properties:
+        source:
+          $ref: ./data-provenance.yaml
+        proposal-disposition:
```

Comment:
This could have been reframed as a boolean (Rejected?), but I started with any enum in case there was another disposition needed in your use cases. I seem to recall wanting to signal that a particular proposal was the most limiting, but we discussed needing to accommodate the possibility of a different proposal for continuous and each emergency duration.

Reply:
I think I'm more confused by Rejected in this case.  If a proposal is always considered we count it used, but then if its rejected, wouldn't that imply that it gets kicked out with an error when its sent?  It feels inconsistent with the 202 Accepted code we send on patch, where I think we're giving the user a guarantee we are at least considering the thing.

Reply:
We agreed this was useful but needed a good description for clarity. That's been added.